macros.in

```diff
@@ -750,11 +750,12 @@ Supplements:   (%{name} = %{version}-%{release} and langpacks-%{1})\
   RPM_SOURCE_DIR=\"%{_sourcedir}\"\
   RPM_BUILD_DIR=\"%{_builddir}\"\
   RPM_OPT_FLAGS=\"%{optflags}\"\
+  RPM_LD_FLAGS=\"%{?build_ldflags}\"\
```

Comment:
I think this would be the first variable we would export which is unset by default in rpm upstream? We probably want to account for that.